quiche/src/h3/mod.rs

```diff
@@ -422,29 +422,86 @@ pub enum Error {
     VersionFallback,
 }
 
+/// Error codes as defined in https://www.rfc-editor.org/rfc/rfc9114.html#http-error-codes.
+pub enum Http3ErrorWireCode {
+    /// No error. This is used when the connection or stream needs to be closed, but there is no error
+    /// to signal.
+    H3NoError = 0x100,
```

Comment:
All these `H3` and `Http3` prefixes are unnecessary, since this is already in the `h3` module. The enum should be just called `WireErrorCode`.